src/components/shared/CallNodeContextMenu.js

```diff
@@ -440,8 +451,12 @@ class CallNodeContextMenuImpl extends React.PureComponent<Props> {
 
     const funcIndex = callNodeTable.func[callNodeIndex];
     const isJS = funcTable.isJS[funcIndex];
+    const hasCategory = callNodeTable.category[callNodeIndex] !== -1;
```

Comment:
optional nit: it could be useful to extract 
```
const categoryIndex = callNodeTable.category[callNodeIndex];
```
and use it both here and for `categoryName`.

Reply:
Good catch.